ember-cli-build.js

```diff
@@ -29,5 +29,11 @@ module.exports = function(defaults) {
   // please specify an object with the list of modules as keys
   // along with the exports of each module as its value.
 
+  app.import('bower_components/blueimp-file-upload/js/vendor/jquery.ui.widget.js');
+  app.import('bower_components/blueimp-file-upload/js/jquery.iframe-transport.js');
+  app.import('bower_components/cloudinary/js/load-image.min.js');
+  app.import('bower_components/cloudinary/js/canvas-to-blob.min.js');
+  app.import('bower_components/cloudinary/js/jquery.cloudinary.js');
+
```

Comment:
For displaying images (no upload) I think we only need to import jquery.cloudinary.js.